frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -154,6 +154,8 @@ class ParameterValueWidget extends Component {
               className={cx(S.parameter, className, {
                 [S.selected]: hasValue,
               })}
+              role="button"
+              aria-label={hasValue ? value : placeholder}
```

Comment:
You can read the reason described in https://github.com/metabase/metabase/pull/29569/files#diff-124fa4dcae5da179c159091f4b0d0e3046144da58ab6ea3efa6741f69e18c2f6R91, why I made it a button.

TODO: Fix the link above.

Reply:
I think the parameter value cannot be used as a label otherwise it would change every time the value changes. Could this change be reverted?